ais_nginx.conf

```diff
@@ -9,13 +9,26 @@ server {
     # the port your site will be served on
     listen      80;
     # the domain name it will serve for
-    server_name niwen2.aramada.nu; # substitute your machine's IP address or FQDN
+    server_name ais2.armada.nu; # substitute your machine's IP address or FQDN
```

Comment:
I've put it in the push script for ais2 now, if it works, ill put it in ais aswell